src/imex.rs

```diff
@@ -785,18 +793,6 @@ async fn export_database(
         .await
 }
 
-/// Sets `Config::BccSelf` (and `DeleteServerAfter` to "never" in effect) if needed so that new
-/// messages are present on the server after a backup restoration or available for all devices in
-/// multi-device case. NB: Calling this after a backup import isn't reliable as we can crash in
-/// between, but this is a problem only for old backups, new backups already have `BccSelf` set if
-/// necessary.
-async fn adjust_bcc_self(context: &Context) -> Result<()> {
-    if context.is_chatmail().await? && !context.config_exists(Config::BccSelf).await? {
```

Comment:
Just to clarify the logic here: `is_chatmail()` is checked because for non-chatmail `BccSelf` is true by default anyway, and `config_exists(Config::BccSelf)` is checked because that time it was decided that it's not good to change settings which the user modified by hand (i think we should really avoid this or at least not do it silently)